apps/website/src/pages/collaborations.tsx

```diff
@@ -28,6 +28,12 @@ type Collaboration = {
 type Collaborations = Record<string, Collaboration>;
 
 const collaborations = {
+  emilyywang: {
```

Comment:
```suggestion
  emilyWang: {
```
Curious on your opinion here, I know we don't really have a set way of doing these?

Reply:
i dont really mind. your suggestion seems more consistent though